torch/quantization/qconfig.py

```diff
@@ -138,3 +138,52 @@ def assert_valid_qconfig(qconfig: Optional[Union[QConfig, QConfigDynamic]],
         )
         assert not is_per_channel, \
             'Per channel weight observer is not supported yet for ConvTranspose{n}d.'
+
+QConfigAny = Union[QConfig,
```

Comment:
this exists in `torch.quantization.fx.qconfig_utils`, could we move it out of fx folder and reuse it instead of creating a new one?

Reply:
yeah, will do